src/emqx_mod_topic_metrics.erl

```diff
@@ -363,7 +363,7 @@ calculate_speed(CurVal, #speed{last_v = LastVal, tick = Tick, acc = Acc, samples
     CurSpeed = (CurVal - LastVal) / ?TICKING_INTERVAL,
 
     %% calculate the average speed in last 5 seconds
-    case Tick =< 5 of
+    case Tick < 5 of
```

Comment:
use a macro for this const. e.g. `AVERAGE_WINDOW_SIZE`.
and the comment line above should be updated too ?